span.go

```diff
@@ -381,7 +399,15 @@ func (s *Span) reportSelfTime() {
 	} else {
 		s.tx.childrenTimer.childEnded(endTime)
 	}
-	s.tx.spanTimings.add(s.Type, s.Subtype, s.Duration-s.childrenTimer.finalDuration(endTime))
+
+	// Collect span timings when dropped and the destination service isn't empty
+	spanDuration := s.Duration - s.childrenTimer.finalDuration(endTime)
+	if s.dropped() && s.IsExitSpan() {
+		dstStvRes := s.Context.destinationService.Resource
```

Comment:
```suggestion
		resource := s.Context.destinationService.Resource
```

nit: the other var name is unpronounceable to me :smile: